Examples/MAX32650/Library_Use/gpiolib.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file    gpiolib.h
```

Comment:
Thanks for the comment but these examples are independent examples.

The purpose of Library_Generate is to demonstrate the step how user create a library.
The purpose of Library_Use is to demonstrate the steps how user use a library that generated by another project.

Making a link to "C:/MaximSDK/Examples/xxx/Library_Generate" not a good idea on my view.

Reply:
Sounds good, to demonstrate usage of  IPATH, PROJ_LDFLAGS too, lets put lib related files under "lib" folder instead of root folder.

@ttmut  please create below folders in Library_Use example
- lib
- lib/include
folders and put .a and .h in these folder, apply related changes on the project.mk files.

Reply:
I could only make this work with PROJ_LIBS and PROJ_LDFLAGS as shown below. If I am to use LIBS, PROJ_LDFLAGS becomes redundant.
```
IPATH += lib/include
PROJ_LDFLAGS += -Llib
PROJ_LIBS += myLib
```

```
IPATH += lib/include
PROJ_LDFLAGS += -Llib #Unnecessary since lib directory is already included in LIBS
LIBS += lib/libmyLib.a
```

Reply:
You're right, I'm not sure why the core makefiles have both `LIBS` and `PROJ_LIBS`.  I think the original intention was `LIBS` would be internal to the SDK.  We could streamline this at some point in the future.

`PROJ_LIBS` is the best choice for this.  Looks good!